src/widget/weffectchainpresetbutton.cpp

```diff
@@ -73,6 +73,7 @@ void WEffectChainPresetButton::populateMenu() {
             tooltip.append("<br/>");
             tooltip.append(effectNames.join("<br/>"));
```

Comment:
I am afraid join() is also a reallocation in a loop as well as append() above.
Clear() below deletes all QStrings in the QList(), keeping only the space for the data-pointer.    
Getting rid of the whole string list would be better or reuse the QStrings in it.

Reply:
I agree, feel free to fix this in a follow-up. At least this should fix the failing CI build after we merged `2.4` to `main`.